src/app/client/src/app/app.component.spec.ts

```diff
@@ -122,6 +122,20 @@ describe('AppComponent', () => {
     expect(document.title).toBe(mockData.tenantSuccess.result.titleName);
     expect(document.querySelector).toHaveBeenCalled();
   });
+  it('Should display the tenant logo if user is not logged in', () => {
+    const userService = TestBed.get(UserService);
```

Comment:
why are you creating the const when you are not using it

Reply:
removed the unused constant